nemo/collections/nlp/data/dialogue_state_tracking_sgd/data_processor.py

```diff
@@ -0,0 +1,675 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
How about changing the name of this folder from dialogue_state_tracking_sgd to just dialogue_state_tracking and then add sgd to the name of the files. Like sgd_data_processor.py? or create a subfolder named sgd so that we may add other models in future.

Reply:
I would like to wait for @ekmb  to finish her review before I refactor,

Reply:
+1 for `dialogue_state_tracking` folder and `sgd` subfolder